lib/project_types/extension/models/specification.rb

```diff
@@ -18,14 +18,16 @@ def self.build(feature_set_attributes)
               .call(identifier, namespace: Features)
               .rescue { OpenStruct }
               .then { |c| c.new(**feature_attributes) }
-              .unwrap { |error| raise error }
+              .unwrap { |error| error ? raise(error) : nil }
```

Comment:
I'm not sure I understand the reason for this change. Will this return nil if there is no error?  Why would we be in that state?

Reply:
That was a left over piece from a debugging session. Good catch. I reverted back to the previous implementation.